src/deploy/osp_deployer/deployer.py

```diff
@@ -215,6 +226,8 @@ def deploy():
         director_vm.node_introspection()
         director_vm.update_sshd_conf()
         director_vm.assign_node_roles()
+        # Should we be calling function below?
+        # director_vm.revert_sshd_conf()?
```

Comment:
Can you remove the above 2 comments?  Gael responded that we should revert the sshd conf.